mkdocs/docs/backends/gpu-vulkan.md

```diff
@@ -1,10 +1,149 @@
+    repository is recommended, as it places Vulkan libraries and tools under
+    system paths so it's easy to discover.
+
+    If the showed version is lower than Vulkan 1.2, you will need to update the
+    driver for your GPU.
+
+
+=== "Windows"
+
+    Run the following command in a shell:
+
+    ``` shell
+    vulkaninfo | grep apiVersion
+    ```
+
+    If `vulkaninfo` does not exist, you will need to [install the latest Vulkan
+    SDK][vulkan-sdk].
+
+    If the showed version is lower than Vulkan 1.2, you will need to update the
+    driver for your GPU.
+
+
+## Get Runtime and Compiler
+
+### Get IREE runtime with Vulkan HAL driver
+
+Next you will need to get an IREE runtime that supports the Vulkan HAL driver
+so it can execute the model on GPU via Vulkan.
+
+<!-- TODO(??): vcpkg -->
+
+#### Build runtime from source
+
+Please make sure you have followed the [Getting started][get-started] page
+to build IREE for Linux/Windows and the [Android cross-compilation][android-cc]
+page for Android. The Vulkan HAL driver is compiled in by default on non-Apple
+platforms.
+
+<!-- TODO(??): a way to verify Vulkan is compiled in and supported -->
+
+If you want to explicitly specify HAL drivers to support, you will need to add
+`Vulkan` to the `IREE_HAL_DRIVERS_TO_BUILD` CMake list variable when
+configuring (for target).
+
+### Get compiler for SPIR-V exchange format
+
+Vulkan expects the program running on GPU to be expressed by the [SPIR-V][spirv]
+binary exchange format, which the model must be compiled into.
+
+<!-- TODO(??): vcpkg -->
+
+#### Build compiler from source
+
+Please make sure you have followed the [Getting started][get-started] page
+to build IREE for Linux/Windows and the [Android cross-compilation][android-cc]
+page for Android. The SPIR-V compiler backend is compiled in by default on all
+platforms.
+
+If you want to explicitly specify HAL drivers to support, you will need to add
+`Vulkan-SPIRV` to the `IREE_TARGET_BACKENDS_TO_BUILD` CMake list variable when
+configuring (for host).
+
+## Compile and Run the Model
+
+With the compiler for SPIR-V and runtime for Vulkan, we can now compile a model
+and run it on the GPU.
+
+### Compile the model
+
+IREE compilers transform a model into its final deployable format in many
+sequential steps. A model authored with Python in an ML framework should use the
+corresponding framework' import tool to convert into a format (i.e.,
+[MLIR][mlir]) expected by main IREE compilers first.
+
+Using MobileNet v2 as an example, you can download the SavedModel with trained
+weights from [TensorFlow Hub][tf-hub-mobilenetv2] and convert it using IREE's
+[TensorFlow importer][tf-import]. Then,
+
+#### Compile using the command-line
+
+In the build directory, run the following command:
+
+``` shell
+iree/tools/iree-translate \
+    -iree-mlir-to-vm-bytecode-module \
+    -iree-hal-target-backends=vulkan-spirv \
+    iree_input.mlir -o mobilenet.vmfb
```

Comment:
Could highlight the `-iree-hal-target-backends=vulkan-spirv \` line

https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#highlighting-specific-lines

```suggestion
``` shell hl_lines="3"
iree/tools/iree-translate \
    -iree-mlir-to-vm-bytecode-module \
    -iree-hal-target-backends=vulkan-spirv \
    iree_input.mlir -o mobilenet.vmfb
```

(I haven't tried that yet, but it could be useful here)

Reply:
ohh, cool!!